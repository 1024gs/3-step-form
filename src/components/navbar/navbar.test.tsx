import React from '../../../vendors/react';
import {shallow} from 'enzyme';
import Navbar from './navbar';

describe('Navbar', () => {
  it('renders', () => {
    const wrapper = shallow(<Navbar />);
    expect(wrapper).toMatchSnapshot();
  });
});
