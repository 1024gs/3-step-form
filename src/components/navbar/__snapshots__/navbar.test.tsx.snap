// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders 1`] = `
<nav
  className="navbar navbar-light"
>
  <div
    className="container"
  >
    <span
      className="navbar-brand"
    >
      Tray.io
    </span>
    <ul
      className="navbar-nav"
    >
      <li
        className="nav-item"
      >
        <a
          className="nav-link"
          href="#"
        >
          Menu
        </a>
      </li>
    </ul>
  </div>
</nav>
`;
